### PR TITLE
DBAAS-1871 : Add a healthcehck notebook

### PR DESCRIPTION
**Pull request by jramineni**

### CODE REVIEW

File: 2DW28WSQJ/note.json [hunk 1]
```diff
@@ -0,0 +1,290 @@
+{
+  "paragraphs": [
+    {
+      "text": "%md\nThis notebook is used to perform check that Zeppelin is functioning properly.\n\u003cB\u003ePLEASE DON NOT DELETE OR MODIFY THIS NOTEBOOK \u003c/B\u003e",
```
Comment (MurrayBrown) [inline 2DW28WSQJ/note.json]: fix typo: DO NOT

File: 2DW28WSQJ/note.json [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "markdown",
+          "editOnDblClick": true,
+          "completionSupport": false
+        },
+        "editorMode": "ace/mode/markdown",
+        "editorHide": false,
```
Comment (MurrayBrown) [inline 2DW28WSQJ/note.json]: editorHide should be true on this paragraph

File: 2DW28WSQJ/note.json [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/python",
+        "title": true
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
```
Comment (MurrayBrown) [inline 2DW28WSQJ/note.json]: results should be cleared in all 'check' paragraphs of this notebook by default, else it gives the impression that something was checked and passed

**Review by MurrayBrown — COMMENTED** [review]
I have a couple of suggested fixes

File: 2DW28WSQJ/note.json (hunk 1 above)
Comment (jramineni) [reply 2DW28WSQJ/note.json]: Done

File: 2DW28WSQJ/note.json (hunk 2 above)
Comment (jramineni) [reply 2DW28WSQJ/note.json]: Done

File: 2DW28WSQJ/note.json (hunk 3 above)
Comment (jramineni) [reply 2DW28WSQJ/note.json]: Done

**Review by tedmonddong — APPROVED** [review]
+1

**Review by MurrayBrown — APPROVED** [review]
i'm loving the notebook now, cool!

**Pull request merged**